js/client-chat.js

```diff
@@ -104,6 +104,7 @@
 			}
 			this.$chatbox.val('');
 			this.$chatbox.trigger('keyup'); // force a resize
```

Comment:
I wonder if this is actually the problematic line.

Reply:
I thought that too. Would keydown work instead?